xopt/configure.py

```diff
@@ -216,7 +203,7 @@ def fill_defaults(dict1, defaults):
     for k, v in defaults.items():
         if k not in dict1:
             dict1[k] = deepcopy(v)
-
+            
```

Comment:
Consider removing the empty spaces here!

Reply:
maybe `black` will save us here

Reply:
deleted